docs/organizations/managing-repositories.md

```diff
@@ -14,29 +16,41 @@ This page lists the repositories that already belong to your organization on Cod
 %}
 
-To add a new repository to Codacy, click the button **Add repository** at the top right-hand corner of the page. This opens a window listing the repositories in your Git provider organization that don't belong to your organization on Codacy yet.
+Depending on your [permissions on the Git provider](roles-and-permissions-for-organizations.md) you can:
+
+-   **Add** a new repository to Codacy to start analyzing it. You need admin permission over the repository to add it to Codacy.
+-   **Follow** a repository that was already added to Codacy by a repository admin.
+
+To add or follow a repository, click the button **Manage repository** at the top right-hand corner of the page. This opens a window listing your organization repositories.
 
-![Adding a repository](images/repositories-add.png)
+<!-- TODO PLUTO-368 Update screenshot -->![Adding a repository](images/repositories-add.png)
 
 !!! important
-    To see your repositories in this list, make sure that you have admin permissions over the repositories on the Git provider and that Codacy has permissions to access the repositories.
+    To see your repositories in this list, make sure that you have permissions over the repositories on the Git provider and that Codacy has permissions to access the repositories.
 
-Add one or multiple repositories to your organization by clicking **Add** next to the repositories. If you have many repositories, you can use the search field above the list to <span class="skip-vale">quickly</span> find a specific repository.
+Add or follow your organization repositories by clicking **Add** or **Follow** next to the repositories. If you have many repositories, you can use the search field above the list to <span class="skip-vale">quickly</span> find a specific repository.
 
-After adding repositories, close the window to return to the list of repositories in your organization. Although Codacy will immediately start analyzing the new repositories, they will display empty metrics until the first analysis returns results.
+Then, close the window to return to your repositories list. Although Codacy will immediately start analyzing new added repositories, they will display empty metrics until the first analysis returns results.
 
 ![Waiting for first analysis results](images/repositories-analyzing.png)
 
+!!! note
+    -   When an **admin user** adds a new repository to Codacy, all users with admin permissions over that repository will start following it.
+
+    -   If you access directly on Codacy to the details of a repository that you have permissions over and you **weren't following yet**, Codacy automatically adds that repository to your repositories list. For example, when you access the repository directly from a link on the Git provider UI.
+
+    -   If you access directly on Codacy to the details of a repository that you **don't have permissions anymore**, Codacy automatically removes that repository from your repositories list.
```

Comment:
Here's my attempt at trying to simplify this explanation a bit, by focusing more on the point of view of the user:

```suggestion
    -   You automatically **start following** a repository as soon as you access any page from that repository. For example, when you access the repository using a direct link on your Git provider UI.

        Conversely, you automatically **stop following** a repository as soon as you try accessing any page from that repository but you don't have permissions to see that repository anymore.
```

Reply:
🤓